docs: re-order Read Replicas doc sections (#21703)
supabase/supabase: apps/docs/content/guides/platform/read-replicas.mdx

---
title: 'Read Replicas'
description: 'Deploy additional read-only databases in multiple regions for your project.'
subtitle: 'Deploy additional read-only databases in multiple regions for your project'
---

> <Admonition type="caution" label="Read Replicas are currently available to organizations on Pro plan and above who have signed up for early access.">

Reviewer: ```suggestion
<Admonition type="note" label="Read Replicas are available to organizations on Pro plan and above who have signed up for early access.">
```

according to our style guide, `caution` is for things that can cause serious user inconvenience (but short of data loss or leak), I'd say this is definitely a `note`

> To sign up for early access, [request access here](https://forms.supabase.com/read-replica).

Reviewer: ```suggestion
[Request early access here](https://forms.supabase.com/read-replica).
```

</Admonition>

<figure className="max-w-[700px] mx-auto">
    <Image
        className="hidden dark:block"

>         alt="Map view of all project databases"

Reviewer: no point in doing the double image if light and dark mode images are the same, you can just put in a single image and remove the className

| ------------ | ------ | ------ | ------ | ------ |
| Primary      | ✅     | ✅     | ✅     | ✅     |
| Read Replica | ✅     | -      | -      | -      |

A database is spun up automatically with every Supabase project. This database is the Primary database. You can choose to add Read Replicas to your project and those Replicas will receive and synchronize data from the Primary database.

Replication is asynchronous and happens in the background so transactions on the Primary aren't blocked. The time delay between writing data to the Primary and the Read Replica receiving the change is replication lag.

Read Replicas have many applications:

- They reduce read load on the Primary database.
  - For example, you can use a Read Replica for complex analytical queries and reserve the Primary for user-facing create, update, and delete operations.
- For projects that need to cover a global audience, additional databases can be deployed closer to users in any of our twelve supported regions to reduce latency.
- Read Replicas provide data redundancy.

## Prerequisites

A project needs to satisfy these requirements to deploy a Read Replica:

1. Running on AWS.

>    - Support for projects on Fly.io will be added at a later date.

Reviewer: ```suggestion
   - Support for projects on Fly.io is coming.
```
